_modules/module-01.md

```diff
@@ -5,6 +5,7 @@ title: Supervised learning
 Sep 6 
 : [Text classification](https://nyu-cs2590.github.io/course-material/fall2023/lecture/lec01/main.pdf) [[recording](https://nyu.zoom.us/rec/share/ILTAqa42w57MczJHzlP-VaEcmdu2cjsA3-NTcpNtmEuD1-diUHI6N8H329VJBurY.WST3O1J7yNzoQMaa)]
   : **HW 1 out**{: .label .label-green }
+  [[zip](https://nyu-cs2590.github.io/fall2023/assignments/hw1_fall2023.zip)]
```

Comment:
Could you rename hw1_fall2023.zip to hw1.zip? It's on fall2023 webpage so by default everything is fall2023.